From f1e63d3d62f21f7c69b9be2e8ecdac5afed51170 Mon Sep 17 00:00:00 2001
From: Ilya Ledvich <ilya@compulab.co.il>
Date: Tue, 21 Apr 2020 15:26:58 +0300
Subject: [PATCH 51/55] ARM: config: compulab: update Moby defconfig

Update compulab_imx7_moby_defconfig.
Set following options:
 - CONFIG_LOCALVERSION="-moby-comp"
Enable following options:
 - CONFIG_USB_CHIPIDEA

Signed-off-by: Ilya Ledvich <ilya@compulab.co.il>
---
 arch/arm/configs/compulab_imx7_moby_defconfig | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm/configs/compulab_imx7_moby_defconfig b/arch/arm/configs/compulab_imx7_moby_defconfig
index 67250b1991a2..5ea06860f0ad 100644
--- a/arch/arm/configs/compulab_imx7_moby_defconfig
+++ b/arch/arm/configs/compulab_imx7_moby_defconfig
@@ -1,3 +1,5 @@
+CONFIG_LOCALVERSION="-moby-comp"
+# CONFIG_LOCALVERSION_AUTO is not set
 CONFIG_KERNEL_LZO=y
 CONFIG_SYSVIPC=y
 CONFIG_POSIX_MQUEUE=y
@@ -524,7 +526,7 @@ CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_EHCI_MXC=y
 CONFIG_USB_ACM=m
 CONFIG_USB_STORAGE=y
-CONFIG_USB_CHIPIDEA=m
+CONFIG_USB_CHIPIDEA=y
 CONFIG_USB_CHIPIDEA_UDC=y
 CONFIG_USB_CHIPIDEA_HOST=y
 CONFIG_USB_SERIAL=m
-- 
2.20.1

